docs/pages/repo/docs/core-concepts/monorepos/_meta.json

```diff
@@ -2,5 +2,6 @@
   "running-tasks": "Running Tasks",
   "task-dependencies": "Task Dependencies",
   "filtering": "Filtering Workspaces",
+  "code-generation": "Code Generation",
```

Comment:
How about "Generating new workspaces"? We aren't generating code so much as places to put code

Reply:
I checked out how this is described in the docs of a few other tools (rails, nx etc) and they all refer to it as code generation so I'm in favor of leaving as is for the cross shoppers. 

And you can generate whatever you want with the custom generators!

Reply:
To be fair, you can indeed generate code with this feature, all the way down to UI components.